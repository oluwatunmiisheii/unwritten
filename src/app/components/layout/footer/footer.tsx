import React from 'react'

export const Footer = () => {
  return (
    <footer className="text-center pb-8 pt-24 text-sm text-[#212529]">
      © {new Date().getFullYear()} <span className="font-pacifico">PhotoSearch.</span> ❤️ Wilson
      Adenuga
    </footer>
  )
}
