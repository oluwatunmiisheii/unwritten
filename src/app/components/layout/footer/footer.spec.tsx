import { Footer } from './footer'
import { expect, test, describe } from 'vitest'
import { render } from '@testing-library/react'

describe('Footer', () => {
  test('should render correctly', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
