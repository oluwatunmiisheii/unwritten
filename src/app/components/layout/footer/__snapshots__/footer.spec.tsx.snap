// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > should render correctly 1`] = `
<div>
  <footer
    class="text-center pb-8 pt-24 text-sm text-[#212529]"
  >
    © 
    2024
     
    <span
      class="font-pacifico"
    >
      PhotoSearch.
    </span>
     ❤️ Wilson Adenuga
  </footer>
</div>
`;
